### ISSUE TITLE
PPOCR-v5 训练自己数据集卡死怎么解决

### ISSUE THREAD

**Issue by fengzhengxiong** [PaddlePaddle/PaddleOCR#15680]
### 🔎 Search before asking

- [x] I have searched the PaddleOCR [Docs](https://paddlepaddle.github.io/PaddleOCR/) and found no similar bug report.
- [x] I have searched the PaddleOCR [Issues](https://github.com/PaddlePaddle/PaddleOCR/issues) and found no similar bug report.
- [x] I have searched the PaddleOCR [Discussions](https://github.com/PaddlePaddle/PaddleOCR/discussions) and found no similar bug report.

### 🐛 Bug (问题描述)

我使用最新的ocr-v5模型，按照官方文档去训练自己的一个数据集，但是它加载模型成功后，一直没有下一步进展了。

### 🏃‍♂️ Environment (运行环境)

conda + python3.10,    cuda：11.8

### 🌰 Minimal Reproducible Example (最小可复现问题的Demo)

下边是我的cmd输出：

D:\Anaconda3\envs\ppocr_env\lib\site-packages\paddle\utils\cpp_extension\extension_utils.py:711: UserWarning: No ccache found. Please be aware that recompiling all source files may be required. You can download and install ccache from: https://github.com/ccache/ccache/blob/master/doc/INSTALL.md
  warnings.warn(warning_message)
[2025/06/11 10:13:39] ppocr WARNING: Skipping import of the encryption module.
[2025/06/11 10:13:39] ppocr INFO: Architecture :
[2025/06/11 10:13:39] ppocr INFO:     Backbone :
[2025/06/11 10:13:39] ppocr INFO:         det : True
[2025/06/11 10:13:39] ppocr INFO:         name : PPHGNetV2_B4
[2025/06/11 10:13:39] ppocr INFO:     Head :
[2025/06/11 10:13:39] ppocr INFO:         k : 50
[2025/06/11 10:13:39] ppocr INFO:         mode : large
[2025/06/11 10:13:39] ppocr INFO:         name : PFHeadLocal
[2025/06/11 10:13:39] ppocr INFO:     Neck :
[2025/06/11 10:13:39] ppocr INFO:         intracl : True
[2025/06/11 10:13:39] ppocr INFO:         name : LKPAN
[2025/06/11 10:13:39] ppocr INFO:         out_channels : 256
[2025/06/11 10:13:39] ppocr INFO:     Transform : None
[2025/06/11 10:13:39] ppocr INFO:     algorithm : DB
[2025/06/11 10:13:39] ppocr INFO:     model_type : det
[2025/06/11 10:13:39] ppocr INFO: Eval :
[2025/06/11 10:13:39] ppocr INFO:     dataset :
[2025/06/11 10:13:39] ppocr INFO:         data_dir : ./ocr_dataset/
[2025/06/11 10:13:39] ppocr INFO:         label_file_list : ['./ocr_dataset/test.txt']
[2025/06/11 10:13:39] ppocr INFO:         name : SimpleDataSet
[2025/06/11 10:13:39] ppocr INFO:         transforms :
[2025/06/11 10:13:39] ppocr INFO:             DecodeImage :
[2025/06/11 10:13:39] ppocr INFO:                 channel_first : False
[2025/06/11 10:13:39] ppocr INFO:                 img_mode : BGR
[2025/06/11 10:13:39] ppocr INFO:             DetLabelEncode : None
[2025/06/11 10:13:39] ppocr INFO:             DetResizeForTest : None
[2025/06/11 10:13:39] ppocr INFO:             NormalizeImage :
[2025/06/11 10:13:39] ppocr INFO:                 mean : [0.485, 0.456, 0.406]
[2025/06/11 10:13:39] ppocr INFO:                 order : hwc
[2025/06/11 10:13:39] ppocr INFO:                 scale : 1./255.
[2025/06/11 10:13:39] ppocr INFO:                 std : [0.229, 0.224, 0.225]
[2025/06/11 10:13:39] ppocr INFO:             ToCHWImage : None
[2025/06/11 10:13:39] ppocr INFO:             KeepKeys :
[2025/06/11 10:13:39] ppocr INFO:                 keep_keys : ['image', 'shape', 'polys', 'ignore_tags']
[2025/06/11 10:13:39] ppocr INFO:     loader :
[2025/06/11 10:13:39] ppocr INFO:         batch_size_per_card : 1
[2025/06/11 10:13:39] ppocr INFO:         drop_last : False
[2025/06/11 10:13:39] ppocr INFO:         num_workers : 2
[2025/06/11 10:13:39] ppocr INFO:         shuffle : False
[2025/06/11 10:13:39] ppocr INFO: Global :
[2025/06/11 10:13:39] ppocr INFO:     cal_metric_during_train : False
[2025/06/11 10:13:39] ppocr INFO:     checkpoints : None
[2025/06/11 10:13:39] ppocr INFO:     debug : False
[2025/06/11 10:13:39] ppocr INFO:     distributed : False
[2025/06/11 10:13:39] ppocr INFO:     epoch_num : 500
[2025/06/11 10:13:39] ppocr INFO:     eval_batch_step : [0, 1500]
[2025/06/11 10:13:39] ppocr INFO:     infer_img : doc/imgs_en/img_10.jpg
[2025/06/11 10:13:39] ppocr INFO:     log_smooth_window : 20
[2025/06/11 10:13:39] ppocr INFO:     model_name : PP-OCRv5_server_det
[2025/06/11 10:13:39] ppocr INFO:     pretrained_model : ./inference_model/PP-OCRv5_server_det_infer/PP-OCRv5_server_det_pretrained.pdparams
[2025/06/11 10:13:39] ppocr INFO:     print_batch_step : 10
[2025/06/11 10:13:39] ppocr INFO:     save_epoch_step : 10
[2025/06/11 10:13:39] ppocr INFO:     save_inference_dir : None
[2025/06/11 10:13:39] ppocr INFO:     save_model_dir : ./output/PP-OCRv5_server_det
[2025/06/11 10:13:39] ppocr INFO:     save_res_path : ./checkpoints/det_db/predicts_db.txt
[2025/06/11 10:13:39] ppocr INFO:     use_gpu : True
[2025/06/11 10:13:39] ppocr INFO:     use_visualdl : False
[2025/06/11 10:13:39] ppocr INFO: Loss :
[2025/06/11 10:13:39] ppocr INFO:     alpha : 5
[2025/06/11 10:13:39] ppocr INFO:     balance_loss : True
[2025/06/11 10:13:39] ppocr INFO:     beta : 10
[2025/06/11 10:13:39] ppocr INFO:     main_loss_type : DiceLoss
[2025/06/11 10:13:39] ppocr INFO:     name : DBLoss
[2025/06/11 10:13:39] ppocr INFO:     ohem_ratio : 3
[2025/06/11 10:13:39] ppocr INFO: Metric :
[2025/06/11 10:13:39] ppocr INFO:     main_indicator : hmean
[2025/06/11 10:13:39] ppocr INFO:     name : DetMetric
[2025/06/11 10:13:39] ppocr INFO: Optimizer :
[2025/06/11 10:13:39] ppocr INFO:     beta1 : 0.9
[2025/06/11 10:13:39] ppocr INFO:     beta2 : 0.999
[2025/06/11 10:13:39] ppocr INFO:     lr :
[2025/06/11 10:13:39] ppocr INFO:         learning_rate : 0.001
[2025/06/11 10:13:39] ppocr INFO:         name : Cosine
[2025/06/11 10:13:39] ppocr INFO:         warmup_epoch : 2
[2025/06/11 10:13:39] ppocr INFO:     name : Adam
[2025/06/11 10:13:39] ppocr INFO:     regularizer :
[2025/06/11 10:13:39] ppocr INFO:         factor : 1e-6
[2025/06/11 10:13:39] ppocr INFO:         name : L2
[2025/06/11 10:13:39] ppocr INFO: PostProcess :
[2025/06/11 10:13:39] ppocr INFO:     box_thresh : 0.6
[2025/06/11 10:13:39] ppocr INFO:     max_candidates : 1000
[2025/06/11 10:13:39] ppocr INFO:     name : DBPostProcess
[2025/06/11 10:13:39] ppocr INFO:     thresh : 0.3
[2025/06/11 10:13:39] ppocr INFO:     unclip_ratio : 1.5
[2025/06/11 10:13:39] ppocr INFO: Train :
[2025/06/11 10:13:39] ppocr INFO:     dataset :
[2025/06/11 10:13:39] ppocr INFO:         data_dir : ./ocr_dataset/
[2025/06/11 10:13:39] ppocr INFO:         label_file_list : ['./ocr_dataset/train.txt']
[2025/06/11 10:13:39] ppocr INFO:         name : SimpleDataSet
[2025/06/11 10:13:39] ppocr INFO:         ratio_list : [1.0]
[2025/06/11 10:13:39] ppocr INFO:         transforms :
[2025/06/11 10:13:39] ppocr INFO:             DecodeImage :
[2025/06/11 10:13:39] ppocr INFO:                 channel_first : False
[2025/06/11 10:13:39] ppocr INFO:                 img_mode : BGR
[2025/06/11 10:13:39] ppocr INFO:             DetLabelEncode : None
[2025/06/11 10:13:39] ppocr INFO:             CopyPaste : None
[2025/06/11 10:13:39] ppocr INFO:             IaaAugment :
[2025/06/11 10:13:39] ppocr INFO:                 augmenter_args :
[2025/06/11 10:13:39] ppocr INFO:                     args :
[2025/06/11 10:13:39] ppocr INFO:                         p : 0.5
[2025/06/11 10:13:39] ppocr INFO:                     type : Fliplr
[2025/06/11 10:13:39] ppocr INFO:                     args :
[2025/06/11 10:13:39] ppocr INFO:                         rotate : [-10, 10]
[2025/06/11 10:13:39] ppocr INFO:                     type : Affine
[2025/06/11 10:13:39] ppocr INFO:                     args :
[2025/06/11 10:13:39] ppocr INFO:                         size : [0.5, 3]
[2025/06/11 10:13:39] ppocr INFO:                     type : Resize
[2025/06/11 10:13:39] ppocr INFO:             EastRandomCropData :
[2025/06/11 10:13:39] ppocr INFO:                 keep_ratio : True
[2025/06/11 10:13:39] ppocr INFO:                 max_tries : 50
[2025/06/11 10:13:39] ppocr INFO:                 size : [640, 640]
[2025/06/11 10:13:39] ppocr INFO:             MakeBorderMap :
[2025/06/11 10:13:39] ppocr INFO:                 shrink_ratio : 0.4
[2025/06/11 10:13:39] ppocr INFO:                 thresh_max : 0.7
[2025/06/11 10:13:39] ppocr INFO:                 thresh_min : 0.3
[2025/06/11 10:13:39] ppocr INFO:                 total_epoch : 500
[2025/06/11 10:13:39] ppocr INFO:             MakeShrinkMap :
[2025/06/11 10:13:39] ppocr INFO:                 min_text_size : 8
[2025/06/11 10:13:39] ppocr INFO:                 shrink_ratio : 0.4
[2025/06/11 10:13:39] ppocr INFO:                 total_epoch : 500
[2025/06/11 10:13:39] ppocr INFO:             NormalizeImage :
[2025/06/11 10:13:39] ppocr INFO:                 mean : [0.485, 0.456, 0.406]
[2025/06/11 10:13:39] ppocr INFO:                 order : hwc
[2025/06/11 10:13:39] ppocr INFO:                 scale : 1./255.
[2025/06/11 10:13:39] ppocr INFO:                 std : [0.229, 0.224, 0.225]
[2025/06/11 10:13:39] ppocr INFO:             ToCHWImage : None
[2025/06/11 10:13:39] ppocr INFO:             KeepKeys :
[2025/06/11 10:13:39] ppocr INFO:                 keep_keys : ['image', 'threshold_map', 'threshold_mask', 'shrink_map', 'shrink_mask']
[2025/06/11 10:13:39] ppocr INFO:     loader :
[2025/06/11 10:13:39] ppocr INFO:         batch_size_per_card : 8
[2025/06/11 10:13:39] ppocr INFO:         drop_last : False
[2025/06/11 10:13:39] ppocr INFO:         num_workers : 8
[2025/06/11 10:13:39] ppocr INFO:         shuffle : True
[2025/06/11 10:13:39] ppocr INFO: profiler_options : None
[2025/06/11 10:13:39] ppocr INFO: train with paddle 3.0.0 and device Place(gpu:0)
[2025/06/11 10:13:39] ppocr INFO: Initialize indexes of datasets:['./ocr_dataset/train.txt']
[2025/06/11 10:13:39] ppocr INFO: Initialize indexes of datasets:['./ocr_dataset/test.txt']
W0611 10:13:39.478636  3916 gpu_resources.cc:119] Please NOTE: device: 0, GPU Compute Capability: 8.6, Driver API Version: 12.6, Runtime API Version: 11.8
W0611 10:13:39.484757  3916 gpu_resources.cc:164] device: 0, cuDNN Version: 8.9.
[2025/06/11 10:13:39] ppocr INFO: train dataloader has 25 iters
[2025/06/11 10:13:39] ppocr INFO: valid dataloader has 50 iters
[2025/06/11 10:13:40] ppocr INFO: load pretrain successful from ./inference_model/PP-OCRv5_server_det_infer/PP-OCRv5_server_det_pretrained
[2025/06/11 10:13:40] ppocr INFO: During the training process, after the 0th iteration, an evaluation is run every 1500 iterations

**Comment by PiggyOverFlow** [comment]
我也遇到了和你一样的情况，请问是怎么解决的，我的运行环境是 CUDA:11.8, RTX-4090 24G ，OS：ubuntu:20.04, Python是3.10的
